pages/glossary/decorative.md

```diff
@@ -3,4 +3,6 @@ title: Decorative
 key: decorative
```

Comment:
@WilcoFiers I have streamlined the applicability, I think you will like it :)

Reply:
I like it now!

I think the rule is correct now.

Reply:
I initially took out "included in the accessibility tree" but added it again because otherwise elements with display:none would be applicable and they shouldn't be.